misc/formatter/modules/formatter-core/src/main/java/org/ballerinalang/formatter/core/FormattingTreeModifier.java

```diff
@@ -1062,8 +1062,10 @@ public ReturnStatementNode transform(ReturnStatementNode returnStatementNode) {
     public FunctionCallExpressionNode transform(FunctionCallExpressionNode functionCallExpressionNode) {
         NameReferenceNode functionName = formatNode(functionCallExpressionNode.functionName(), 0, 0);
         Token functionCallOpenPara = formatToken(functionCallExpressionNode.openParenToken(), 0, 0);
+        indent(2);
```

Comment:
I guess, value to has a meaning here. Shall we move this to a constant and add a proper Java doc? Replace all the usages of this.

Reply:
Added as a formatting options